sys-libs/fts-standalone/fts-standalone-1.2.5.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Standalone fts library for use with musl"
+HOMEPAGE="https://github.com/pullmoll/musl-fts"
+SRC_URI="https://github.com/pullmoll/musl-fts/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~mips ~ppc ~x86"
+IUSE="static-libs"
+
+DEPEND="
+	!sys-libs/glibc
+	!sys-libs/uclibc
+"
+
+S="${WORKDIR}/musl-fts-${PV}"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable static-libs static)
+}
+
+src_install() {
+	default
+	find "${D}" -name '*.la' -delete || die
+	insinto /usr/lib/pkgconfig
+	newins musl-fts.pc fts-standalone.pc
```

Comment:
We are renaming the package from musl-fts.pc to fts-standalone.pc so that's not something that can go into the build system.  lib -> $(libdir) si correct, but your pull request to musl-fts will only make things harder for us.